content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -8,7 +8,7 @@ weight: 30
 
 <!-- overview -->
 
-<img src="https://raw.githubusercontent.com/kubernetes/kubeadm/master/logos/stacked/color/kubeadm-stacked-color.png" align="right" width="150px">
+<img src="/images/kubeadm-stacked-color.png" align="right" width="150px"></img>
```

Comment:
I don't think so, removing the end tag will result in the following codes being parsed as html, and thus markdown links not rendered

Reply:
@kbhawkey worth pointing out, if you add a line return after `<img>` it will still break the links.

Hence, I think the best way is still adding the end tag `</img>`. I've also tried the self-closing `<img ... />` tag but it does not seem to work either

Reply:
👍